clearwater.mk

```diff
@@ -1,15 +1,24 @@
-DEB_COMPONENT := clearwater-monit
-DEB_MAJOR_VERSION := 5.8${DEB_VERSION_QUALIFIER}
-DEB_NAMES := clearwater-monit
+PKG_COMPONENT := clearwater-monit
+PKG_MAJOR_VERSION := 5.8.1
+PKG_NAMES := clearwater-monit
 
 include build-infra/cw-deb.mk
+include build-infra/cw-rpm.mk
+
+.PHONY: configure
+configure:
+	./bootstrap && ./configure
 
 .PHONY: deb-build-monit
-deb-build-monit:
+deb-build-monit: configure
 	echo -e "${DEB_COMPONENT} (${DEB_MAJOR_VERSION}-${DEB_MINOR_VERSION}) unstable; urgency=low\n" >debian/changelog
```

Comment:
I've had a look and see that cw-deb.mk will create the DEB_* variables, but given that we create PKG_* at the top of this file, i'm torn on what we should echo here. 
I don't mind really what we goo for, but it feels slightly simpler/more consistent to change this to PKG_*. If this wont work, then fine.